api/data_explorer/controllers/export_url_controller.py

```diff
@@ -103,6 +107,22 @@ def _get_entities_dict(cohort_name, query):
                 'query': query
             }
         })
+
+    for doc in doc_generator:
+        for sample in doc.get('samples', []):
+            sample_id = sample['sample_id']
+            export_sample = {
+                k.replace('.', '_'): v
```

Comment:
Let's use "Has_phase1_chrMT_SNPs" instead of "verily-public-data_human_genome_variants_1000_genomes_sample_info_Has_phase1_chrMT_SNPs" for column name

Reply:
Done (indexer PR).